classes/sync-classes/class-customer-sync.php

```diff
@@ -252,6 +252,7 @@ public static function get_customer_by_order_id( $order_id ) {
 		$billing_country    = $order->get_billing_country();
 		$meta_data          = $order->get_meta_data();
 		$billing_address    = self::get_street_address( $order );
+		$date_joined        = $order->get_date_created();
```

Comment:
Should we have `getOffsetTimestamp()` here as well?

Reply:
I don't see why? Here we just pass order data as customer data for non-registered users and the other fields are not modified in any way.

Later this same oject will be used to get the customer's properties for non-registered user and at that point the same date which is set here gets converted to a date which is sent to custobar.

Reply:
True, didn't follow the code close enough.